src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java

```diff
@@ -230,6 +208,9 @@ private void initializeHttpPoolingClient() {
   public void setMaximumPoolingConnections(int maximumPoolingConnections) {
     if (maximumPoolingConnections < 1)
       throw new IllegalArgumentException("maximumPoolingConnections must be 1 or greater");
+    if(httpPoolingClient!=null){
```

Comment:
Shouldn't we have space after the "if" and around the != and after the ")"?  "if (httpPoolingClient != null) {"